guides/making_new_layers_and_models_via_subclassing.py

```diff
@@ -235,11 +235,16 @@ def __init__(self, rate=1e-2):
         self.rate = rate
 
     def call(self, inputs):
-        self.add_loss(self.rate * tf.reduce_sum(inputs))
+        self.add_loss(self.rate * tf.reduce_mean(inputs))
         return inputs
 
 
 """
+Notice that `add_loss()` can take the result of plain TensorFlow operations.
+There is no need to call a `Loss` object here. (In fact, you could *not* use
+the convenient default reduction by batch size, because that is not allowed
+under a distribution strategy.)
```

Comment:
This sentence will not be understood by 99% of readers of this guide:

"(In fact, you could *not* use
the convenient default reduction by batch size, because that is not allowed
under a distribution strategy.)"

Either remove the sentence or clarify it.

Reply:
I'd like to clarify, and I think it would help if I could link to the documentation of `Loss` that forbids SUM_OVER_BATCH_SIZE.

tensorflow.org has it [here](https://www.tensorflow.org/api_docs/python/tf/keras/losses/Loss#:~:text=Using%20'AUTO'%20or%20'SUM_OVER_BATCH_SIZE'%20will%20raise%20an%20error), but where does keras.io have it?

Reply:
No advice on the `Loss` docstring? Then I just remove the extra reference.